rest-api/participant_enums.py

```diff
@@ -68,6 +68,11 @@ class SampleStatus(messages.Enum):
   QNS_FOR_PROCESSING = 17
   QUALITY_ISSUE = 18
 
+class EhrStatus(messages.Enum):
+  """Status of EHRs"""
+  MISSING = 0
```

Comment:
I think MISSING could be misinterpreted in this context. Perhaps `NOT_PRESENT` ?